TypeSystem/Reflection/golden/smoke_test_struct.json

```diff
@@ -0,0 +1,143 @@
+{
+  "structs": [
+    [
+      9201385900795340000,
+      {
+        "type_id": 9201385900795340000,
+        "name": "FullTest",
+        "super_type_id": 0,
+        "fields": [
+          [
+            9209518734145112000,
+            "primitives"
+          ],
+          [
+            9319767778871345000,
+            "v1"
+          ],
+          [
+            9315917652032240000,
+            "v2"
+          ],
+          [
+            9331586039661578000,
+            "p"
+          ],
+          [
+            9213438387000307000,
+            "o"
+          ]
+        ]
+      }
+    ],
+    [
+      9209518734145112000,
+      {
+        "type_id": 9209518734145112000,
+        "name": "Primitives",
+        "super_type_id": 0,
+        "fields": [
+          [
+            9000000000000000000,
```

Comment:
Test fails on both - OS X and Ubuntu - with correct TypeIDs in `Actual`.
Looks like you just committed wrong file.